### PR TITLE
fsck: moved to linux, macOS version created

### LINKED ISSUE #1764: fsck is different on macOS X
```
fsck usage: fsck [-fdnypq] [-l number]
```

https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man8/fsck.8.html

### PR DESCRIPTION
**Pull request by gingerbeardman**
Closes #1764

### CODE REVIEW

File: pages/osx/fsck.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# fsck
+
+> Check the integrity of a filesystem or repair it. The filesystem should be unmounted at the time the command is run. fsck is now essentially a wrapper that fsck_hfs, fsck_apfs, fsck_msdos, fsck_exfat, and fsck_udf as needed.
```
Comment (agnivade) [inline pages/osx/fsck.md]: Is the 3rd sentence absolutely essential ? The heading is a bit too long IMO for a tldr page.

**Comment by agnivade** [comment]
Hey @gingerbeardman - I actually wanted to remove the sentence altogether if it is not necessary. Moving it to a new line does not help much. But if you feel that it is indeed required, we can keep it. The grammar of the sentence is also slightly incorrect.

**Comment by gingerbeardman** [comment]
I feel it's needed @agnivade

File: pages/osx/fsck.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# fsck
+
+> Check the integrity of a filesystem or repair it. The filesystem should be unmounted at the time the command is run.
+> So fsck is now essentially a wrapper that calls fsck_hfs, fsck_apfs, fsck_msdos, fsck_exfat, and fsck_udf as needed.
```
Comment (sbrl) [inline pages/osx/fsck.md]: This feels awkwardly worded. Perhaps `fsck is a wrapper that calls`... instead?

Comment (agnivade) [reply pages/osx/fsck.md]: I had the same feeling. Agree with @sbrl's suggestion.

Comment (gingerbeardman) [reply pages/osx/fsck.md]: Feel free to do it, I deleted the repo thinking everything was done, sorry!

Comment (agnivade) [reply pages/osx/fsck.md]: Uh oh. Since you deleted the fork, we will have to close this PR, send a new PR with the changes.

**Comment by gingerbeardman** [comment]
I give up

**Comment by agnivade** [comment]
No worries. The new PR is over here  https://github.com/tldr-pages/tldr/pull/1803

**Comment by waldyrious** [comment]
Hey @gingerbeardman, I'm just now seeing this. Your comments above, and in #1953 ("that was a very painful process") clearly show that something is not working well in the tldr-pages contribution workflow. First of all, I'm sorry to hear you had a hard time. I'd love if you could let us know in what ways you think that the process could be improved. It's sometimes hard for us to fully grasp the perspective of a new contributor to the project, so your insight would be much appreciated!

**Comment by gingerbeardman** [comment]
In a nutshell: with the tldr repo there is a lot of talk and not much action by the maintainers. For me, maintainers should do more than just speculate or direct how things should be done, they should be in with their sleeves rolled up getting things done! Surely? We're open source developers here, not open source paper pushers.

My PR was a simple issue that got out of control with the input of agnivade, who I have found difficult to deal with on a number of occasions (with all due respect), and some other maintainers. Eventually I just walked away as it simply was not worth my time and effort. I just wanted to add a quick command example not take a degree in github/travis/etc! This is by far the most difficult experience I've had with OSS, and I've contributed to many including the well known RetroArch project and have a number of fairly popular repositories of my own.

For tldr, a collection of small text files, there are way too many rules and not enough simple organisation. The extra hoops like travis and grammar rules people have to jump through are just insane. Way OTT. I'm sure they were fun to set up, and theoretically they should help but they do not. It would be better if you let people submit stuff, and then cleaned up any "missing trailing period" or whatever, as that's not really important compared to the actual command information. If the maintainers spent as much time actually maintaining the contents of the repo, rather than poking holes in PRs and blowing any small issue into a long drawn out process.

People like me arrive with the best of intentions, to write a quick example for a command they needed some help with, to make everybody's lives easier, and they are met with a metaphorical stack of paperwork and bureaucracy that quickly zaps any interest they had. I have stopped by a few times and every time I see multiple examples in issues and PRs. The structure of the repo is for the maintainers to deal with, IMHO, not the passerby who wants to add a quick command or example.

A simple goal might be to accept PRs with minimal friction? Accepting whatever people add and then clean it up after the fact. This will help you see what people actually want to add, rather than setting a whole bunch of rules that make the PR process very painful.

**Pull request closed without merging**